cloudant-client/src/test/java/com/cloudant/tests/IndexTests.java

```diff
@@ -22,9 +22,9 @@
 import com.cloudant.client.api.CloudantClient;
 import com.cloudant.client.api.Database;
 import com.cloudant.client.api.model.FindByIndexOptions;
-import com.cloudant.client.api.model.Index;
 import com.cloudant.client.api.model.IndexField;
-import com.cloudant.client.api.model.IndexField.SortOrder;
+import com.cloudant.client.api.query.JsonIndex;

```

Comment:
Not related but this file appears to have incorrect line endings (probably some ancient legacy...)

Reply:
Fixed in 7034ae0